src/systems/physics/Physics.cc

```diff
@@ -1298,6 +1338,99 @@ void PhysicsPrivate::UpdatePhysics(EntityComponentManager &_ecm)
           return true;
         }
 
+        auto posLimits = _ecm.Component<components::JointPositionLimitsCmd>(
+            _entity);
+        if (posLimits && !posLimits->Data().empty())
+        {
+          auto limits = posLimits->Data();
```

Comment:
nit: can this be a const ref since `limits` is not going to change?

Reply:
fixed in all 3 places